frontend/src/components/funding/FundingAgreementsTab.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <v-container fluid class="pa-0">
+    <FundingSearchCard :loading="loading" @search="loadFundingAgreements" />
+    <h2 class="mt-8 mb-2">Funding Details</h2>
+    <v-skeleton-loader :loading="loading" type="table-tbody">
+      <v-data-table :headers="headers" :items="fundingAgreements" item-key="guid" :items-per-page="10" density="compact" :mobile="null" mobile-breakpoint="md" class="soft-outline">
+        <template #[`item.startDate`]="{ item }">
+          {{ format.formatDate(item?.startDate) }}
+        </template>
+        <template #[`item.endDate`]="{ item }">
+          {{ format.formatDate(item?.endDate) }}
+        </template>
+        <template #[`item.status`]="{ item }">
+          <span :class="getStatusClass(item?.statusCode)" class="pt-1 pb-1 pl-2 pr-2">{{ item?.statusName }}</span>
+        </template>
+        <template #[`item.actions`]="{ item }">
+          <v-btn variant="text" @click="$router.push({ name: 'funding', params: { fundingGuid: item.fundingId } })">
+            <v-icon v-if="item?.statusCode === FUNDING_AGREEMENT_STATUS_CODES.SIGNATURE_PENDING" size="large">mdi-signature-freehand</v-icon>
+            <v-icon v-else-if="[FUNDING_AGREEMENT_STATUS_CODES.SUBMITTED, FUNDING_AGREEMENT_STATUS_CODES.ACTIVE].includes(item?.statusCode)" size="large">mdi-folder-open-outline</v-icon>
+          </v-btn>
+        </template>
+      </v-data-table>
+    </v-skeleton-loader>
+  </v-container>
+</template>
+
+<script>
+import FundingSearchCard from '@/components/funding/FundingSearchCard.vue'
+import alertMixin from '@/mixins/alertMixin.js'
+import FundingAgreementService from '@/services/fundingAgreementService'
+import { FUNDING_AGREEMENT_STATUS_CODES } from '@/utils/constants'
+import format from '@/utils/format'
+
+// TODO: Remove this constant after we have Payment Type for FA in CRM
+const PAYMENT_FILTER_TYPE_VALUES = {
```

Comment:
This can be removed. Value is always 'Base Funding'.

Reply:
it's updated